controllers/k8ssandra/datacenters.go

```diff
@@ -466,3 +459,13 @@ func dcUpgradePriority(dc api.CassandraDatacenterTemplate) int {
 
 	return 2
 }
+
+func (r *K8ssandraClusterReconciler) removeRebuildDcAnnotation(kc *api.K8ssandraCluster, ctx context.Context) result.ReconcileResult {
+	patch := client.MergeFromWithOptions(kc.DeepCopy())
```

Comment:
MergeFromWithOptions, but no options? Is this meant to be optimistic lock patch?

Reply:
I don't think so, I'll change this to `MergeFrom()`.